cyberphoria/Elevator-Simulation
# Improve select algorithm

- added back BoundedBufferTest (oops)
- added StatusUpdate
- moved elevatorNumber to SystemEvent
- add makeStatusUpdate() method to Elevator
- update ElevatorMonitor with StatusUpdate
- remove StatusRequest and StatusResponse as they are no longer needed
- fix a minor error in the inputs.json 6 Up 2 to 2 Up 6
